lib/extensions/postgres_cdc_rls/migrations.ex

```diff
@@ -1,10 +1,73 @@
+    CreateRealtimeCheckEqualityOpFunction,
+    CreateRealtimeBuildPreparedStatementSqlFunction,
+    CreateRealtimeCastFunction,
+    CreateRealtimeIsVisibleThroughFiltersFunction,
+    CreateRealtimeApplyRlsFunction,
+    GrantRealtimeUsageToAuthenticatedRole,
+    EnableRealtimeApplyRlsFunctionPostgrest9Compatibility,
+    UpdateRealtimeSubscriptionCheckFiltersFunctionSecurity,
+    UpdateRealtimeBuildPreparedStatementSqlFunctionForCompatibilityWithAllTypes,
+    EnableGenericSubscriptionClaims,
+    AddWalPayloadOnErrorsInApplyRlsFunction,
+    UpdateChangeTimestampToIso8601ZuluFormat,
+    UpdateSubscriptionCheckFiltersFunctionDynamicTableName,
+    UpdateApplyRlsFunctionToApplyIso8601,
+    AddQuotedRegtypesSupport,
+    AddOutputForDataLessThanEqual64BytesWhenPayloadTooLarge,
+    AddQuotedRegtypesBackwardCompatibilitySupport,
+    RecreateRealtimeBuildPreparedStatementSqlFunction,
+    NullPassesFiltersRecreateIsVisibleThroughFilters,
+    UpdateApplyRlsFunctionToPassThroughDeleteEventsOnFilter,
+    MillisecondPrecisionForWalrus,
+    AddInOpToFilters,
+    EnableFilteringOnDeleteRecord
+  }
+
   alias Realtime.Helpers, as: H
 
+  @migrations [
+    {20_211_116_024_918, CreateRealtimeSubscriptionTable},
+    {20_211_116_045_059, CreateRealtimeCheckFiltersTrigger},
+    {20_211_116_050_929, CreateRealtimeQuoteWal2jsonFunction},
+    {20_211_116_051_442, CreateRealtimeCheckEqualityOpFunction},
+    {20_211_116_212_300, CreateRealtimeBuildPreparedStatementSqlFunction},
+    {20_211_116_213_355, CreateRealtimeCastFunction},
+    {20_211_116_213_934, CreateRealtimeIsVisibleThroughFiltersFunction},
+    {20_211_116_214_523, CreateRealtimeApplyRlsFunction},
+    {20_211_122_062_447, GrantRealtimeUsageToAuthenticatedRole},
+    {20_211_124_070_109, EnableRealtimeApplyRlsFunctionPostgrest9Compatibility},
+    {20_211_202_204_204, UpdateRealtimeSubscriptionCheckFiltersFunctionSecurity},
+    {20_211_202_204_605,
+     UpdateRealtimeBuildPreparedStatementSqlFunctionForCompatibilityWithAllTypes},
+    {20_211_210_212_804, EnableGenericSubscriptionClaims},
+    {20_211_228_014_915, AddWalPayloadOnErrorsInApplyRlsFunction},
+    {20_220_107_221_237, UpdateChangeTimestampToIso8601ZuluFormat},
+    {20_220_228_202_821, UpdateSubscriptionCheckFiltersFunctionDynamicTableName},
+    {20_220_312_004_840, UpdateApplyRlsFunctionToApplyIso8601},
+    {20_220_603_231_003, AddQuotedRegtypesSupport},
+    {20_220_603_232_444, AddOutputForDataLessThanEqual64BytesWhenPayloadTooLarge},
+    {20_220_615_214_548, AddQuotedRegtypesBackwardCompatibilitySupport},
+    {20_220_712_093_339, RecreateRealtimeBuildPreparedStatementSqlFunction},
+    {20_220_908_172_859, NullPassesFiltersRecreateIsVisibleThroughFilters},
+    {20_220_916_233_421, UpdateApplyRlsFunctionToPassThroughDeleteEventsOnFilter},
+    {20_230_119_133_233, MillisecondPrecisionForWalrus},
+    {20_230_128_025_114, AddInOpToFilters},
+    {20_230_128_025_212, EnableFilteringOnDeleteRecord}
+  ]
```

Comment:
Is there a way to do this automatically? It looks like a place where easy make mistakes when adding a new migration

Reply:
there's no straightforward way to do this. let's get this in and then i'll try and figure out a better way for this later.